sdk/messaging/azservicebus/receiver_test.go

```diff
@@ -752,6 +755,38 @@ func TestReceiverMultiTopic(t *testing.T) {
 	require.Equal(t, []string{"sent to other queue2"}, getSortedBodies(otherMessages))
 }
 
+func TestReceiverMessageLockExpires(t *testing.T) {
+	client, cleanup, queueName := setupLiveTest(t, &admin.QueueProperties{
+		LockDuration: to.Ptr("PT5S"),
+	})
+	defer cleanup()
+
+	sender, err := client.NewSender(queueName, nil)
+	require.NoError(t, err)
+
+	err = sender.SendMessage(context.Background(), &Message{Body: []byte("hello")}, nil)
+	require.NoError(t, err)
+
+	receiver, err := client.NewReceiverForQueue(queueName, nil)
+	require.NoError(t, err)
+
+	messages, err := receiver.ReceiveMessages(context.Background(), 1, nil)
+	require.NoError(t, err)
+
+	// sleep so our message locks expire
+	time.Sleep(6 * time.Second)
```

Comment:
Can you skip this in playback?

Reply:
I wish! This is actually AMQP underneath, so no recordings.